2018-10-25-Ticha-Review

```diff
@@ -0,0 +1,50 @@
+Lillehaugen, Brook Danielle, George Aaron Broadwell, Michel R. Oudijk, & Laurie Allen. 2015.  
+*Ticha: a digital text explorer for Colonial Zapotec*.  
+First edition.(http://ticha.haverford.edu/)
+
+Ticha means language in Zapotec, an indigenous language spoken in Southern Mexico. Thicha is a digital platform  
+that was and is being created by an interdisciplinary team of scholars from different fields in the United States  
+and Mexico at Haverford College Libraries: Brook D. Lillehaugen, George A. Broadwell, Michel R. Oudijk, and Laurie Allen.  
+Ticha analyzes data through two main technological platforms: FLEx which is Fieldworks Language Explorer, a system for  
```

Comment:
This sentence is a little confusing - it's unclear what the two platforms are. Perhaps break it into two sentences? I might use a different word in terms of platform. Perhaps "Ticha encodes the data in two ways:"